lib/Api.php

```diff
@@ -82,4 +84,63 @@ public static function registerHooks()
             );
         }, 10, 2);
     }
+
+    public static function registerBlocks($layouts)
+    {
+        // create block and field group for block
+        if (!function_exists('acf_register_block_type')) return null;
```

Comment:
I think this check could be done right before useGutenberg is defined. If the function does not exist it should just disable gutenberg again, will fix later.